processing/data_collection/gazette/spiders/sp_itu.py

```diff
@@ -0,0 +1,53 @@
+import dateparser
+import chompjs
+import base64
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpItuSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3523909"
+    PDF_URL = "https://dosp.com.br/exibe_do.php?i={}"
+
+    GAZETTE_API_URL = "https://www.dosp.com.br/api/index.php/dioe.js/4923"
+
+    allowed_domains = ["dosp.com.br"]
+    name = "sp_itu"
+
+    def start_requests(self):
+        yield Request(self.GAZETTE_API_URL)
+
+    def parse(self, response):
+
+        response_js = chompjs.parse_js_object(response.text)
+
+        for element in response_js.get("data"):
+            print("DATE")
+            date = self.extract_date(element)
+
+            print("URL")
+            url = self.extract_url(element)
+            print(url)
+
+            yield Gazette(
+                date=date,
+                file_urls=[url],
+                territory_id=self.TERRITORY_ID,
+                power="executive_legislature",
+                scraped_at=datetime.utcnow(),
+            )
```

Comment:
I think we can get the `edition_number` field from `"edicao_do"` too. Don't you think?